backend/impl/build.gradle.kts

```diff
@@ -7,6 +7,7 @@ dependencies {
   api(project(":backend:api"))
   api(project(":branchLayout:api"))
 
+  testImplementation(project(":frontend:file"))
```

Comment:
Hmmm backend depending on frontend (even if for the test)... looks questionable... lemme see how that dependency is used, however...

Reply:
Huh is that dependency really needed tho? 🤔

Reply:
Yeah, project builds without that, assuming it's some artifact from draft-ness of the PR